ds3-cli-certification/src/main/java/com/spectralogic/ds3cli/certification/CertificationUtil.java

```diff
@@ -0,0 +1,168 @@
+            LOG.info("Could not delete job Id {}", jobId);
+        }
+    }
+
+    public static void cancelTapeEject(final Ds3Client client, final String barcode) {
+        try {
+            client.cancelEjectTapeSpectraS3(new CancelEjectTapeSpectraS3Request(barcode));
+        } catch (final IOException e) {
+            LOG.info("Failed cancel eject tape, barcode {}", barcode);
+        }
+    }
+
+    public static String getBucketName(final String testName) {
+        return "test_" + testName.replaceAll("[ !,.:;<>&]+", "_");
+    }
+
+    public static UUID getValidTapePartition(final Ds3Client client) throws IOException {
+        final GetTapePartitionsSpectraS3Request getTapePartitions = new GetTapePartitionsSpectraS3Request();
+        final GetTapePartitionsSpectraS3Response getTapePartitionsResponse = client.getTapePartitionsSpectraS3(getTapePartitions);
+        return getTapePartitionsResponse.getTapePartitionListResult().getTapePartitions().get(0).getId();
+    }
+
+    public static boolean waitForTapePartitionQuiescedState(
+            final Ds3Client client,
+            final UUID tapePartitionId,
+            final Quiesced quiescedState) throws InterruptedException, IOException {
+        int retries = 0;
+        final int max_retries = 12;
+        while (quiescedState != client.getTapePartitionSpectraS3(new GetTapePartitionSpectraS3Request(tapePartitionId.toString())).getTapePartitionResult().getQuiesced()) {
+            LOG.info("Sleeping 5 minutes while waiting for tape partition {} to change to Quiesced state {}...", tapePartitionId, quiescedState);
+            sleep(300 * 1000); // sleep 5 minutes
+            if (++retries > max_retries) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    public static boolean ensureTapePartitionQuiescedState(
+            final Ds3Client client,
+            final UUID tapePartitionId,
+            final Quiesced desiredState) throws IOException, InterruptedException {
+        if (desiredState == client.getTapePartitionSpectraS3(new GetTapePartitionSpectraS3Request(tapePartitionId.toString())).getTapePartitionResult().getQuiesced()) {
+            return true;
+        }
+
+        try {
+            if (desiredState == Quiesced.NO) {
+                client.modifyAllTapePartitionsSpectraS3(new ModifyAllTapePartitionsSpectraS3Request(Quiesced.NO));
+            } else {
+                // Must transition state from NO -> PENDING -> YES
+                client.modifyAllTapePartitionsSpectraS3(new ModifyAllTapePartitionsSpectraS3Request(Quiesced.PENDING));
+                client.modifyAllTapePartitionsSpectraS3(new ModifyAllTapePartitionsSpectraS3Request(Quiesced.YES));
+            }
+        } catch (final FailedRequestException fre) {
+            // Ignore Request failure if tape partitions are already quiesced
+            LOG.info("Failed to modify TapePartition {} to Quiesce state: {}", tapePartitionId, fre.getMessage());
```

Comment:
Similar comment as the one above about the format of this error print statement.  This should be updated to:

```java

LOG.error("Failed to modify TapePartition " + tapePartitionId + " to Quiesce state: " + fre.getMessage(), e);
```